cast/tests/e2e/helpers/fixtures.rs

```diff
@@ -0,0 +1,75 @@
+use crate::helpers::constants::{
+    ACCOUNT, ACCOUNT_FILE_PATH, CASM_CONTRACT_PATH, NETWORK, SIERRA_CONTRACT_PATH, URL,
+};
+use camino::Utf8PathBuf;
+use cast::{get_account, get_network, get_provider};
+use starknet::accounts::{Account, SingleOwnerAccount};
```

Comment:
did you try to construct raw requests? were there any problems?
I'm just afraid that we may be overly relying on starknet-rs, and wonder if we could somehow mitigate that

Reply:
I've tried but there is still an issue with signing. We have to have an account to sign those transactions :/